README.md

```diff
@@ -1,2 +1,4 @@
 # feature-branch-workflow
-A simple, boring repo to demonstrate Git workflows involving feature branches 2nd ver
\ No newline at end of file
+A simple, boring repo to demonstrate Git workflows involving feature branches 2nd ver
+
+Bock bock
```

Comment:
Always with the Bock Bock phrase,  leoluna287? Get a new catchphrase!!!!

Reply:
Bock Bock Bock